quantum/keymap_extras/keymap_us_international.h

```diff
@@ -72,7 +72,7 @@
 #define US_K    KC_K    // K
 #define US_L    KC_L    // L
 #define US_SCLN KC_SCLN // ;
-#define US_QUOT KC_QUOT // ' (dead)
+#define US_ACUT KC_QUOT // ´ (dead)
```

Comment:
```suggestion
#define US_DACU KC_QUOT // ´ (dead)
```

Reply:
Hmm - actually there seems to be some confusion on this.  
DACU is not actually "dead acute accent" - it's double-acute accent (in the bepo, croatian, czech, hungarian, romanian, serbian_latin, slovak, and slovenian maps).

Reply:
I was thinking where a layout defines both dead and non-dead acute accents, but not double, `DACU` and `ACUT` could be used. But there is some possible precedent in the ABNT2 header to use `NDAC` instead.